package.json

```diff
@@ -5,12 +5,14 @@
   "dependencies": {
     "ajv": "6.5.4",
     "chart.js": "2.7.2",
+    "highcharts": "^6.2.0",
```

Comment:
Not sure whether to mess with this caret or not. This is what the react-jsx-highcharts instructions said to install.

Reply:
@todo